src/openMVG/features/feature.hpp

```diff
@@ -83,6 +83,11 @@ class SIOPointFeature : public PointFeature {
   float orientation() const { return _orientation; }
   float& orientation() { return _orientation; }
 
+  Vec2f getOrientationVector() const
```

Comment:
maybe call this `getScaledOrientationVector()` and have a `getOrientationVector()` that returns only the unit vector of the orientation:
```c++
Vec2f getOrientationVector() const
{
  return Vec2f(std::cos(orientation()), std::sin(orientation()));
}
Vec2f getScaledOrientationVector() const
{
  return scale()*getOrientationVector();
}
```